starter_code/queries.md

```diff
@@ -4,80 +4,97 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+Filter {name: {\$eq: "Babelgum"}}
+Project {name: 1, \_id:0}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+Filter {number_of_employees: { \$gte: 5000}}
+Sort { number_of_employees: 1}
+limit 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fileds.
 
-<!-- Your Code Goes Here -->
+Filter {founded_year:{$lte:2005, $gte:2000 }}
+Project {name: 1, founded_year:1, \_id:0}
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
-<!-- Your Code Goes Here -->
+Filter {"ipo.valuation_amount": {$gt: 100000000}, founded_year: {$lt: 2010}}
+Project {name: 1, ipo:1, \_id:0}
 
 ### 5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
 
-<!-- Your Code Goes Here -->
+Filter {number_of_employees: {$lt: 1000}, founded_year: {$lt: 2005}}
+Sort {number_of_employees: 1}
 
 ### 6. All the companies that don't include the `partners` field.
 
-<!-- Your Code Goes Here -->
+Filter {partners: { \$exists: false}}
```

Comment:
This only shows if the line partners exists or not but you want to see if it's filled or empty. 
Here is how you can do:
```
{partners: {$eq: []}}
```